src/libasr/codegen/asr_to_wasm.cpp

```diff
@@ -1437,6 +1657,81 @@ class ASRToWASMVisitor : public ASR::BaseVisitor<ASRToWASMVisitor> {
         }
     }
 
+    void visit_ComplexBinOp(const ASR::ComplexBinOp_t &x) {
+        if (x.m_value) {
+            this->visit_expr(*x.m_value);
+            return;
+        }
+        this->visit_expr(*x.m_left);
+        this->visit_expr(*x.m_right);
+        LCOMPILERS_ASSERT(ASRUtils::is_complex(*x.m_type));
+        int a_kind = ASR::down_cast<ASR::Complex_t>(ASRUtils::type_get_past_pointer(x.m_type))->m_kind;
+        switch (x.m_op) {
+            case ASR::binopType::Add: {
+                if (a_kind == 4) {
+                    if (m_self_func_name_idx_map.find("add_c32") == m_self_func_name_idx_map.end()) {
+                        m_self_func_name_idx_map["add_c32"] = no_of_types++;
+                    }
```

Comment:
This just becomes a quick check, like `if (m_self_complex_fn_present[wasm_add_c32])`.